### ISSUE TITLE
Is IE 11 supported? Demo doesn't render correctly

### ISSUE THREAD

**Issue by JRGranell** [pradel/react-responsive-modal#200]
## Environment

| Tech                   | Version |
| ---------------------- | ------- |
| react-responsive-modal |    3.3.0    |
| React                  |     15.6.2    |
| browser                |    IE 11    |

Hi,

We're currently using your library and are required to support IE 11. We noticed a CSS issue when trying to centre the modal. In addition when using IE11 / Win 7 in Virtual box we reproduced the issue using your demo site. 

![virtualbox_ie11 - win7_31_08_2018_11_29_55](https://user-images.githubusercontent.com/9625257/44907983-cd95dc80-ad11-11e8-90f4-224bcb647fc9.png)

I started to look into it and found that removing `justify-content: center;` for the overlay CSS fixed it IE11 - but had to divert to other tasks so hasn't been tested and should only be considered a starting point.

Thanks,

**Comment by JRGranell** [comment]
Hi, tried `flex-direction: column` but didn't make a difference. Looks like just removing `justify-content: center;` has worked. Tested on basic browsers (details in PR) but worth checking if `justify-content: center;` was specifically added for another reason (to avoid regression).


I've found this a useful resource for testing on older versions of IE https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/

Let me know how you get on,

Thanks,
